libbeat/docs/index.asciidoc

```diff
@@ -28,6 +28,8 @@ include::./dashboards.asciidoc[]
 
 include::./newbeat.asciidoc[]
 
+include::./config-file-format.asciidoc[]
```

Comment:
Talking about 'generic enough', there are some sample from filebeat and metricbeat.